app/src/services/httpService.js

```diff
@@ -30,27 +33,18 @@ export class HttpService  {
     }
 
     return fetch(url)
-      .then(this.handleErrors)
-      .then(res => this.makeResponse(res))
-      .catch(err => Promise.reject(err));
-  }
-
-  //Adding this to handle an otherwise unhandled exception, but this will need to be re-worked soon!
-  static handleErrors(response) {
-    if (!response.ok) {
-      throw Error(response.statusText);
-    }
-    return response;
+        .then(res => this.makeResponse(res))
   }
 
   static makeResponse(res) {
     if (res.status === 204) {
         return null;
     }
     if (res.ok) {
-        return res.json();
+      return res.json();
     } else {
-      console.error('HttpService response not OK, cannot return JSON');
+      store.dispatch(notifyActions.showNotify(res.statusText));
+      console.error('Error from Rest Service ' + res.statusText + ',' + res.status);
```

Comment:
you won't need both the dispatch AND the console log once you install the Redux chrome browser extension :)

Reply:
to be clear, even if you haven't yet installed, it... it's probably best to remove the console.error() now